deployment/ansible/roles/model-my-watershed.beaver/templates/spark-jobserver.conf.j2

```diff
@@ -1,4 +0,0 @@
-[/var/log/upstart/spark-jobserver.log]
-multiline_regex_before = ^\s+
-type: spark-jobserver
-tags: spark-jobserver,beaver
```

Comment:
I think we'd want to create it in `/var/log/upstart` and also have beaver configured to ship the logs off the instance

https://github.com/WikiWatershed/model-my-watershed/tree/1a836bb5f8281b7d7f2b0bedb2a2f7bf9131794a/deployment/ansible/roles/model-my-watershed.beaver/templates